discord/discord.go

```diff
@@ -0,0 +1,132 @@
+package discord
+
+import (
+	"fmt"
+	"github.com/bwmarrin/discordgo"
+	"github.com/devanbenz/bits-and-bytes-bot/models"
+	"log"
+	"log/slog"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+)
```

Comment:
I think for imports it would be cool to order them like the following 

```
import (
    "stdlib"

    "3rd party"

    "internal"
)
```

With the idea that we can add - https://golangci-lint.run/usage/linters/#goimports happy to add this to the issue we have open in for 💯